src/OpenLoco/Graphics/Gfx.cpp

```diff
@@ -56,7 +56,7 @@ namespace OpenLoco::Gfx
     static loco_global<uint32_t[147], 0x050B8C8> _paletteToG1Offset;
 
     static loco_global<uint16_t, 0x112C824> _currentFontFlags;
-    static loco_global<int16_t, 0x112C876> _currentFontSpriteBase;
+    static loco_global<int32_t, 0x0112C876> _currentFontSpriteBase;
```

Comment:
in some places this was int32_t and some others was int16_t. it *looks* like int16_t? but I'm unsure, @duncanspumpkin any idea?

Reply:
it should be `uint16_t`.
`0x0112C878` is used as a local variable in some dropdown function.

Reply:
`uint16_t`? the values it is always compared against/assigned to are `int16_t`: https://github.com/OpenLoco/OpenLoco/blob/b2caeb64a3f9d6226fb6427bb1a7ee723ded39b1/src/OpenLoco/Localisation/StringManager.h#L19-L28

Reply:
Oh yeah forgot about the m1/2 settings. int16_t

Reply:
cool, I guessed that would be it and updated the code already